drivers/uktty/ns16550/Config.uk

```diff
@@ -2,6 +2,7 @@ menuconfig LIBUKTTY_NS16550
 	bool "NS16550 / 8250"
 	default n
 	depends on ARCH_ARM_64
+	select LIBUKBOOT
```

Comment:
I am not fully getting why the dependency is declared here and not in the previous commit where the header of libukboot is included that causes that dependency. By mistake? 🤔

Reply:
Most likely accidentally squashed to the wrong commit. Will fix 👍🏼